src/Lattice.h.Rt

```diff
@@ -39,7 +39,7 @@ const int maxSnaps=33;
 class Lattice {
 private:
   Model_m my_model;
-  LatticeContainer * container; ///< Main Container object
+  CartLatticeExecutor* executor; ///< Main execution context, including the lattice container, data, etc.
```

Comment:
Change from pointer to object.

Reply:
Closed in b18a64d39c6de22061dc02e4c5ce95c649d98731